docs/src/optimisation_output_plotting.md

```diff
@@ -72,3 +72,85 @@ When using the `calibrate_model_multistart` function to fit a parameter set, sev
 When plotting a multi start output using the `:objective`, `:best_objective`, or `:parallel_coordinates` plot types, if the number of runs are large, it can sometimes be hard to distinguish information from the plot. Hence, for these plot types, only the `10` runs with the best final objective values are plotted. This can be modified through the `best_idxs_n` optional argument. This is an `Int64`, what number of runs to add to the plot (starting with the best one). Alternatively, the `idxs` optional argument can be used to give the indexes of the runs to plot.
 
 For the `:waterfall`, `:runtime_eval` plot types, by default all runs are plotted. However, if desired, the `best_idxs_n` and `idxs` arguments can be used for these plot types as well.
+
+## Plots comparing the fitted model to the measurements
+After the model has been fitted, it can be useful to compare it to the measurements. This is possible by supplying both the optimisation solution, and the `PEtabModel` used, to the plot command. By default, it will plot, for the first condition, the output solution for all observables. However, any subset of observables can be selected (using the `observable_ids` option). It is also possible to select the condition using the `condition_id` option.
```

Comment:
After the model has been fitted, it is often useful to compare it to the measurements. This is possible by supplying both the optimisation solution, and the `PEtabModel` used, to the plot command. By default, it will plot, for the first simulation condition, the output solution for all observables. However, any subset of observables can be selected (using the `observable_ids` option). It is also possible to select the simulation condition using the `condition_id` option.

Reply:
I will fix this when updating docs later